common/lib/xmodule/xmodule/library_root_xblock.py

```diff
@@ -71,7 +79,7 @@ def render_children(self, context, fragment, can_reorder=False, can_add=False):
         for child_key in children_to_show:  # pylint: disable=E1101
             child = self.runtime.get_block(child_key)
             child_view_name = StudioEditableModule.get_preview_view_name(child)
-            rendered_child = self.runtime.render_child(child, child_view_name, context)
+            rendered_child = self.render_child(child, child_view_name, context)
```

Comment:
I'm not sure how I feel about this code... It calls `render_child` but sometime's we're not actually rendering the child at all. It also requires creating a copy of `context` and injecting a studio template parameter into the XBlock context argument.

I think I'd prefer to see something like:
```python
if self.show_children_previews:
    self.runtime.render_child(child, child_view_name, context)  # No modifications to this method
else:
    self.runtime.render_child_placeholder(child)
```